src/Sparrow/Logging/RavenLogManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NLog;
+
+namespace Sparrow.Logging;
+
+internal class RavenLogManager
+{
+    public static readonly RavenLogManager Instance = new();
+
+    public bool IsAuditEnabled;
+
+    private RavenLogManager()
+    {
+        var innerLogger = LogManager.GetLogger("Audit");
+        IsAuditEnabled = innerLogger.IsInfoEnabled;
```

Comment:
No, it is a separate rule without the ability to change that via settings.json, you can only turn audit on or off and it automatically is set to Info. They only way to override it would be via NLog.config.xml, then we are fully relying on what s in the file.

Reply:
ok